src/main/java/tds/content/TdsContentServiceApplication.java

```diff
@@ -1,4 +1,4 @@
-package org.opentestsystem.delivery;
+package tds.content;
```

Comment:
I wish we had used what you had previously except add content at the end. Or tds.content. However I imagine you are just being consistent with the other TDS apps